### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dariia-romanova**
[DEMO LINK](https://dariia-romanova.github.io/react_static-list-of-todos/)

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1 +1,19 @@
...
+ul,
+h2 {
+  margin: 0;
+  padding: 0;
+}
+
+.App {
+  padding: 40px;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #ecedf1;
```
Comment (Vitaliy-0) [inline src/App.scss:13]: Use variable for color.

File: src/components/TodoList/TodoList.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      todos.map(todo => (
+        <li key={todo.id} className="list__item">
+          <TodoInfo todo={todo} />
+          {
+            todo.user
+              ? (
+                <UserInfo user={todo.user} />
+              )
+              : ''
+          }
```
Comment (Vitaliy-0) [inline src/components/TodoList/TodoList.tsx:23]:
```suggestion
          {
            todo.user && <UserInfo user={todo.user} />
          }
```

Comment (Vitaliy-0) [reply src/components/TodoList/TodoList.tsx:23]: U can write less

File: src/components/UserInfo/UserInfo.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.user {
+  display: flex;
+  gap: 30px;
```
Comment (Vitaliy-0) [inline src/components/UserInfo/UserInfo.scss:3]: Be careful with gap :)

**Review by Vitaliy-0 — APPROVED** [review]
Good job 💯

File: src/components/TodoList/TodoList.tsx (hunk 2 above)
Comment (illiarerun) [reply src/components/TodoList/TodoList.tsx:23]: Aggre with Vitaliy on this one

File: src/components/TodoInfo/TodoInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    { todo.completed
+      ? (
+        <div className="todo__status todo__status--completed">
+          Completed
+        </div>
+      ) : (
+        <div className="todo__status">
+          Not completed
+        </div>
+      )}
```
Comment (illiarerun) [inline src/components/TodoInfo/TodoInfo.tsx:21]: So this divs are almost the same, only classes and inner text are different. So you can use classNames library for setting your --completed modifier and move your ternary inside just to set text, or better move it to variable

**Review by illiarerun — CHANGES_REQUESTED** [review]
Good job 🔥 Let's work on fixes now:

**Review by illiarerun — APPROVED** [review]
Great job 🔥

**Pull request closed without merging**